geoportailv3/static/js/draw/featurepopupservice.js

```diff
@@ -159,6 +163,8 @@ app.FeaturePopup.prototype.show = function(feature, opt_anchor) {
   goog.array.forEach(headers, function(element) {
     this.setDraggable(angular.element(element));
   }, this);
+  // Enable the dropdown menu
+  $('.dropdown-toggle')['dropdown']();
```

Comment:
Actually, the reason why it doesn't work is because we use `stopEvent` for the overlay.
Calling `dropdown` on the element manually after the overlay is created seem to let new events to propagate through the map.

However, I would restrict this to the dropdowns in the overlay:
```javascript
this.element_.find('[data-toggle=dropdown]')['dropdown']();
```

Finally, it would be much pretty if `dropdown` was exposed as an extern in ngeo. Do you want me to create a pull request?